gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -245,6 +245,7 @@ typedef struct
 {
   unsigned int num_tensors; /**< The number of tensors */
   GstTensorInfo info[NNS_TENSOR_SIZE_LIMIT]; /**< The list of tensor info */
+  GstTensorInfo *extra; /**< Info for tensors whose idx is larger than NNS_TENSOR_SIZE_LIMIT */
```

Comment:
How about use index instead of idx?
and..
I think 'list' is a good idea.
because, we already use it as below.
GstTensorInfo info[NNS_TENSOR_SIZE_LIMIT]; /**< The list of tensor

Reply:
It's been applied in previous commits.